jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/list/CaseInstanceListPresenter.java

```diff
@@ -75,10 +95,19 @@ public void createCaseInstance() {
         newCaseInstancePresenter.show();
     }
 
-    protected void refreshData() {
+    private void caseInstancesServiceCall(int currentPage) {
         caseService.call((List<CaseInstanceSummary> cases) -> {
-            view.setCaseInstanceList(cases);
-        }).getCaseInstances(view.getCaseInstanceSearchRequest());
+            visibleCaseInstances.addAll(cases);
+            view.hideLoadButton(cases.size() < PAGE_SIZE);
+            view.setCaseInstanceList(visibleCaseInstances.stream().collect(toList()));
+        }).getCaseInstances(view.getCaseInstanceSearchRequest(),
+                            currentPage,
+                            PAGE_SIZE);
+    }
+
+    protected void refreshData() {
+        visibleCaseInstances.clear();
```

Comment:
This is causing the UI to only show the current page items, because there is no pagination/pages in the UI, we should just keep adding items to the view as they are loaded.